network-container/README.md

```diff
@@ -10,4 +10,24 @@ Next steps:
    You can also do this manually with the following command: `cfn-cli generate`
 3. Implement your resource handlers by adding code to provision your resources in your resource handler's methods.
 
-Please don't modify files `model.go` and `main.go`, as they will be automatically overwritten.
\ No newline at end of file
+Please don't modify files `model.go` and `main.go`, as they will be automatically overwritten.
+
+## Description
+This resource allows you to create, read, update and delete a network container
```

Comment:
This resource allows you to create, read, update and delete a network container.  A network container must be created before one can enable network peering.   You need one container per AWS region that will be peered with.